packages/frontend/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function (app) {
```

Comment:
Don't we need to import this file somewhere ?

Reply:
Apparently not! 

From the [docs](https://create-react-app.dev/docs/proxying-api-requests-in-development/)

> Note: You do not need to import this file anywhere. It is automatically registered when you start the development server.